File: .ci/examples/py-examples/checkavro.py
from pulsar.schema import *

class Student(Record):
  def __init__(self, name, age, grade):
    self.name = name
    self.age = age
    self.grade = grade

  name = String()
  age = Integer()
  grade = Integer()

def process(params):
  params.age = params.age + 1
  return params

File: .ci/examples/node-examples/checkavro.js
function process(params) {
  params['age']['int'] = params['age']['int'] + 1
  return params
}

const definitions = {
  name: 'Student',
  type: 'record',
  fields: [
    {name: 'name', type: ["null", "string"]},
    {name: 'age', type: ["null", "int"]},
    {name: 'grade', type: ["null", "int"]}
  ]
}

module.exports.definitions = definitions
